extensions/vscode/src/views/contentRecords.ts

```diff
@@ -82,10 +82,22 @@ export class ContentRecordsTreeDataProvider
       // 200 - success
       // 500 - internal server error
       const api = await useApi();
-      const response = await api.contentRecords.getAll({
+      const getAllPromise = api.contentRecords.getAll({
         dir: ".",
         recursive: true,
       });
+
+      window.withProgress(
+        {
+          title: "Initializing",
+          location: { viewId: Views.ContentRecords },
+        },
+        async () => {
+          return getAllPromise;
+        },
```

Comment:
Similar question to the last PR wondering if we need to wrap this.

Reply:
API is expecting a function returned.